src/components/CategoryList/index.js

```diff
@@ -2,13 +2,21 @@ import React, { Fragment } from 'react';
 import CategoryItem from './CategoryItem.js';
 import './scss/Style.scss';
 
+type PropsShape = {
+  data?: Array,
+  itemRenderer?: any,
+  clickComponent?: any,
+  baseClassName?: string,
+  isOpenByDefault?: boolean,
+};
+
 const CategoryList = ({
   data = [],
   itemRenderer = null,
```

Comment:
good idea to remove the 'default' values here, so we don't define them in two places. lets stick to standard `Component.defaultProps = {}`